Libs/Core/ctkUtils.cpp

```diff
@@ -143,12 +160,22 @@ QString ctk::extensionToRegExp(const QString& extension)
 {
   // typically *.jpg
   QRegExp extensionExtractor("\\*\\.(\\w+)");
+#if QT_VERSION >= 0x060000
+  int pos = extension.indexOf(extensionExtractor);
+#else
   int pos = extensionExtractor.indexIn(extension);
+#endif
   if (pos < 0)
     {
     return QString();
     }
-  return ".*\\." + extensionExtractor.cap(1) + "?$";
+#if QT_VERSION >= 0x060000
+  QStringList captured = extensionExtractor.namedCaptureGroups();
+  QString cap = captured[1];
```

Comment:
add a check that there are at least 2 strings in captured before trying to access the captured[1]